### ISSUE TITLE
[Feature] Support dark mode

### ISSUE THREAD

**Issue by abhayjatindoshi** [abhayjatindoshi/spendings-client#1]
Need to support dark mode for all the angular components that are used. 
Additionally if possible giving a settings modal to switch between both the UI modes.

**Comment by hiteshpr** [comment]
Can I work on this?

**Comment by abhayjatindoshi** [comment]
@hiteshpr Do let me know if you need any help setting up. :)

**Comment by abhayjatindoshi** [comment]
@hiteshpr Are you still working on this ?

**Comment by hiteshpr** [comment]
Hey, yes I am working on this

**Comment by hiteshpr** [comment]
How to point to APIs? Where are APIs located?

**Comment by abhayjatindoshi** [comment]
Check out the steps below to setup.
1. Create a directory `Spendings`
2. Inside the directory clone `spendings-client` and `spendings-server` as given in the below folder structure
3. create a directory properties and file production.properties
```properties
server.domain=http://localhost:8888/

db.host=localhost:8889
db.username=root
db.password=
db.name=spendings

google.client.id={client-id}
google.client.secret={client-secret}
```
4. Inside the server directory run `sh build.sh`
5. A build folder will be created. Use a PHP server like MAMP or WAMP or XAMPP on htdocs folder in build folder. 


You will have a folder structure similar as given below
```
Spendings
├── build                                 <-- Automatically created using build.sh
│  └─ htdocs                              <-- use it to run PHP server
├── spendings-server                      <-- clone spendings-server repository   
│  └── properties                         <-- create a properties folder in server
|      └─ production.properties           <-- use it to run PHP server
└── spendings-client                      <-- cloned spendings-client repository    

```

After this complete setup you should be able to just run npm start in client directory and use port 4200 to work on client.

**Comment by hiteshpr** [comment]
Hey @abhayjatindoshi Why don't you host the APIs on any free services like Heroku?
It would be much easier for everyone to contribute

**Comment by abhayjatindoshi** [comment]
Yes, Sure I can do that for you. Setting heroku up would take some time. Also I'll check other possibilities. Meanwhile can you try setting up the server as given in the above steps ?
